spec/support/db.js

```diff
@@ -4,7 +4,7 @@ const MongoClient = require('mongodb');
 
 function start() {
   return new MongoMemoryServer({
-    binary: { version: '3.4.6' },
+    binary: { version: '3.6.13' },
```

Comment:
It'd be cool if we could run tests against multiple different mongo versions (I know mongoist does this)

Reply:
True! That'd be nice.